### PR TITLE
MEPTS-1374 |  PrEP NEW - Diff FGH DB

### PR DESCRIPTION
**Pull request by ItsMurumba**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/PrepNewQueries.java [hunk 1]
```diff
@@ -212,54 +213,84 @@ public static String getPatientAgeBasedOnPrepStartDate(
...
+            + "                                                  ON p.patient_id = "
+            + "                                                     e.patient_id "
+            + "                                          INNER JOIN obs o "
+            + "                                                  ON e.encounter_id "
+            + "                                                     = "
+            + "                                                     o.encounter_id "
+            + "                                                      WHERE  p.voided = 0 "
+            + "                                                             AND e.voided = 0 "
+            + "                                                             AND o.voided = 0 "
+            + "                                                             AND "
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/PrepNewQueries.java]: @ItsMurumba please format the code

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
@ItsMurumba Please format the query

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/PrepNewQueries.java (hunk 1 above)
Comment (ItsMurumba) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/PrepNewQueries.java]: I formatted the query using the tool you shared with me @psmatsinhe

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**